src/App.js

```diff
@@ -12,6 +12,7 @@ import {
 } from 'material-ui/Table';
 import { connect } from 'react-redux';
 import { isUrl, httpGet } from './fetch/fetchHelpers';
+import { ADD_SITE } from './actions/site';
```

Comment:
Cool! But the name in capital case uses for constants, not for actions. Please use camelCase for actions, something like `addSite`.